src/components/GenreList.js

```diff
@@ -0,0 +1,36 @@
+
+import React, { useState } from "react";
+import '../GenreList.css';
+
+const GenreBtn = ({genre, isSelected, onGenreSelection}) => {
+    return (
+        <li>
+            <button className={"genre-item " + (isSelected ? "selected" : "")} onClick={() => onGenreSelection(genre)} >{genre} {isSelected}</button>
+        </li>
+    )
+}
+
+const GenreList = () => {
+    const [selectedGenre, setSelectedGenre] = useState('');
```

Comment:
@MykhailoSheliahov good point, it was not taken into consideration in a global scope, so it will be added in module 3 or later when all components will work with each other